classroom.rb

```diff
@@ -0,0 +1,13 @@
+class Classroom
+  attr_accessor :label, :students
+
+  def initialize(label)
+    @label = label
+    @students = []
+  end
+
+  def add_student(student)
+    @students << student
+    student.classroom = self
+  end
```

Comment:
- [ ] Great job for adding a method that serves as 'has-many'(classroom has many students) :+1: Now let's ensure that a student belongs to the classroom by adding a method that serves that purpose in the **Student** class. **Hints:** Kindly check out the Animal class using this [link](https://github.com/microverseinc/curriculum-ruby/blob/main/oop/articles/oop_relationships_by_examples.md#both-ways-relationships)